updatedUsers.ts

```diff
@@ -52,10 +49,26 @@ const users: User[] =
             age: 20,
             physicalAddress: "Dubai"
         }
+    ];
+
+    function myProfile(userId: number): User | undefined {
+        return users.find(user => user.id === userId);
+    }
+
+    function lessAge(ageLimit: number): User | undefined {
+        return users.find(user => user.age < ageLimit);
```

Comment:
No need to let define an ageLimit we already gave you that, which is 27.
No need to pass that as a parameter.